docs/profiles.rst

```diff
@@ -19,7 +19,7 @@ We are collecting the following information for each Rep:
  * Display Name
  * Mozillians Profile
  * Private Email
- * Bugzilla Email
+ * Mozillians Identity Provider e-mail
```

Comment:
The term  `Mozillians Identity Provider` may be confused with an Identity Provider service which provides identities to users. Maybe we should replace this with something that communicates that we mean the identity that is linked in mozillians.org. Unless you mean something completely different here. Thoughts?

Reply:
Do you mean for this instance here only or generally?

Henrik and I came up with that name and though it was specific enough and didn't have any other idea.

Reply:
Had a conversation with @akatsoulas ... 
We believe that the term `Mozilla Login Identity` would currently be the best functional description.

-Henrik

Reply:
OK; will change it. Thanks!